doc/ug/bazel_notes.md

```diff
@@ -272,6 +272,17 @@ test --test_tag_filters=-cw310
 See the [`.bazelrc`](https://github.com/lowRISC/opentitan/blob/master/.bazelrc) file in the OpenTitan repository for more examples.
 Additionally, for more information see the Bazel [documentation](https://bazel.build/run/bazelrc).
 
+### Site-specific `.bazelrc-site` options
```

Comment:
@timothytrippel: What do you think of this text?

Reply:
I think the first paragraph should define what you mean by "site" :)

I've been assuming it's synonymous with "fork" — is that accurate?

Reply:
barring @dmcardle 's suggestsions, LGTM. Thanks for adding this!

Reply:
Thank you for adding that paragraph that defines "compute site"! (Sweet acronym, by the way.)

Reply:
NP.  Thanks for the suggestion.